src/components/Collection/CollectionDescription.js

```diff
@@ -1,48 +1,41 @@
-import React, { Component } from 'react';
-import PropTypes from 'prop-types';
-import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import React, { useState } from "react";
+import PropTypes from "prop-types";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 
 const styles = {
   moreLess: {
-    margin: '0 5px'
+    margin: "0 5px"
   }
 };
 
-class CollectionDescription extends Component {
-  static propTypes = {
-    description: PropTypes.array
-  };
-
-  state = {
-    expanded: false
-  };
+const CollectionDescription = ({ description }) => {
+  const [expanded, setExpanded] = useState();
+  const tooLong = description.length > 1;
 
-  handleClick = e => {
+  const handleClick = e => {
     e.preventDefault();
-    this.setState({ expanded: !this.state.expanded });
+    setExpanded(!expanded);
   };
 
-  render() {
-    const { expanded } = this.state;
-    const { description } = this.props;
-    const tooLong = description.length > 1;
+  return (
+    <>
+      {tooLong && !expanded && description[0]}
```

Comment:
Can this be description.length > 0 ?

Reply:
@katdivyareddy10 that bit of code was added a while ago, and you may bring up a good point.  I'll make a note to look at it and remember what it was/is doing.